src/components/CodeManagement/index.jsx

```diff
@@ -65,6 +80,11 @@ class CodeManagement extends React.Component {
         }
       });
     }
+    // reset location state for paginated coupon orders.
+    history.replace({
```

Comment:
I think I understand why this is here, but do you mind clarifying in more detail? Does this make sense to do on expand or on collapse of coupons?

Reply:
This is when we collapse a coupon details.

Idea is when a coupon is expanded, the pagination or any other query param in url should be removed ... (this is done to minimise ambiguity between two tables) ... but I am open to ideas.